tutorials/temporal_analysis.ipynb

```diff
@@ -0,0 +1,384 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.errorbar(np.subtract(mjd,mjd[0]),flux,yerr=flux_err,fmt=\"o\")    \n",
```

Comment:
Create a `LightCurve` and use it's methods to create a quick-look plot instead.
(if the methods there are broken or insufficient - improve that class).